app/assets/javascripts/sequence_show.js

```diff
@@ -1,7 +1,10 @@
 function init_sequence_show(data) {
 
     // set up the fancy tree
-    initLineageTree(data.tree);
+    initD3TreeView(data.tree, "#lineage-tree");
```

Comment:
Did you change this by accident? If not, did you test it? I don't think this will work by simply replacing this line...


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3460) by @bmesuere on Mon Nov 09 2015 at 20:50._